curl/pkg.yaml

```diff
@@ -27,7 +28,8 @@ steps:
             --enable-threaded-resolver \
             --with-ca-bundle=${TOOLCHAIN}/etc/ssl/certs/ca-certificates \
             --with-zlib \
-            --with-ssl
+            --with-ssl \
+            --without-libpsl
```

Comment:
Any particular reason?

Reply:
Fair point. I wonder if anyone implemented "cookie" like protection with redirects without checking for user to have browser.